cluster.go

```diff
@@ -75,6 +96,8 @@ func NewClusterProfile(cfg *ClusterConfig) (*ClusterProfile, error) {
 }
 
 func (m *ClusterProfile) deployHelm(testName string) error {
+	//nolint
```

Comment:
necessary? is it for errcheck?

Reply:
Yeah, for the linter, no need to check those errors.